refinery/tool_manager/management/commands/load_tools.py

```diff
@@ -41,6 +41,12 @@ def add_arguments(self, parser):
             help='Overwrite ToolDefinitions with the same names instead of '
                  'skipping their creation'
         )
+        parser.add_argument(
+            '--branch',
+            action='store',
+            dest='branch',
+            help='Branch from the registry to target for Vis tool installation'
+        )
```

Comment:
Maybe make the "master" default explicit here, rather than the init? The help text will be more explicit, and you can get rid of one `if`